### ISSUE TITLE
'Busy days' request

### ISSUE THREAD

**Issue by git-a-21** [GCalToolkit/GCalPlus#2]
I find the busy days feature helpful, but red doesn't stand out properly for me. The date text is pretty small, so I was wondering if you could have a toggle feature of showing a circle around the date or something to draw the distinction. And/or have the text bolded but keep it black.
![GCal busy](https://user-images.githubusercontent.com/97137557/153287420-607f3e9a-78ff-4696-a0ca-97740dd41594.jpg)
![GCal busy upgrade](https://user-images.githubusercontent.com/97137557/153972171-565e4da5-8ba2-4a1a-9755-43e5765e4c50.png)

**Comment by GCalToolkit** [comment]
Thanks for the suggestion, I'll see what I can do but this might take some time as it will require adding additional options so that users who are happy with the current display are not affected.

**Comment by git-a-21** [comment]
Of course. Maybe as a toggle feature, red as default but with some other options?
